synthtool/gcp/templates/node_library/.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,15 +6,20 @@ about: Create a report to help us improve
 
 Thanks for stopping by to let us know something could be better!
 
-**PLEASE READ**: If you have a support contract with Google, please create an issue in the [support console](https://cloud.google.com/support/) instead of filing on GitHub. This will ensure a timely response.
+**PLEASE READ**
 
-Please run down the following list and make sure you've tried the usual "quick fixes":
+1) Is this a client library issue or a product issue?
+This is the client library for {{metadata['product']}}. We will only be able to assist with issues that pertain to the behaviors of this library. If the issue you're experiencing is due to the behavior of the product itself, please visit [{{metadata['product']}} Support page]({{metadata['product-url']}}/docs/getting-support) to reach the most relevant engineers.
 
+2) Did someone already solve this?
   - Search the issues already opened: https://github.com/{{metadata['repository']}}/issues
   - Search the issues on our "catch-all" repository: https://github.com/googleapis/google-cloud-node
-  - Search StackOverflow: http://stackoverflow.com/questions/tagged/google-cloud-platform+node.js
+  - Search or ask on StackOverflow (engineers monitor these tags): http://stackoverflow.com/questions/tagged/google-cloud-platform+node.js
 
-If you are still having issues, please be sure to include as much information as possible:
+3) Do you have a support contract?
```

Comment:
I know I said above to keep that bit about support, and I still think we should!  It's ok to say it twice.